tools/ci_build/github/azure-pipelines/linux-ort-srv-nightly-pipeline.yml

```diff
@@ -0,0 +1,40 @@
+jobs:
```

Comment:
I guess we don't need a separate nightly pipeline. I disabled the current one for PR validation and scheduled it to run at 1 AM every day.

Reply:
The current CI build for onnx runtime server only takes about 10 min in total but it could catch any API changes, especially we are using internal APIs which may change more frequently than external APIs. So I would still prefer keep the CI build and run the nightly build with more tests.

Reply:
Using ~10 min CI build to make sure internal API change will not impact the server code. The ROI looks good to me. :-) Please re-enable it and would not impact much overall.

Reply:
Do you think we need this if we are moving to official cxx api?

Reply:
We definitely need to consider move to official APIs when they are ready. It will save the team's maintenance cost overall.